iml-docker.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=iml-docker
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+StandardError=null
+StandardOutput=null
+WorkingDirectory=/etc/iml-docker
+ExecStart=docker stack deploy -c /etc/iml-docker/docker-compose.yml -c /etc/iml-docker/docker-compose.overrides.yml iml --resolve-image=never
+# @FIXME: Need to poll service removal since docker stack rm is async
```

Comment:
This should work:
```
ExecStop=docker stack rm iml && while [ "$(docker stack ps iml 2>&1 | grep 'nothing found in stack') == 1" ]; do echo "Waiting on docker stack to finish removing" && sleep 1; done;
```

Reply:
I don't think we want to echo anything during service shutdown.

Reply:
hrrm, we could also look at the exit code (which returns 1 when the stack is empty). Do we want to do that instead of grepping?

Reply:
Yeah I think that should work.

Reply:
My concern is there could be other scenarios where `docker stack ps iml` could return 1. 

On the flip side if `docker stack ps iml` ever changes it's text when there are no services running, then this will loop forever.

Reply:
Ended up going with exit code.